### PR TITLE
zed: yoga merge

### PR DESCRIPTION
**Pull request by markgoddard**
- CloudKitty fix for ES->OS migration
- Document the manual process to delete a broken LB
- docs: a couple of bits for the RL9 migration
- Update Consul and Vault to latest MPL releases
- Handle undefined Ansible variable
- Fix growroot playbook for disks ending with p
- Add release note

### CODE REVIEW

File: releasenotes/notes/cloudkitty-fix-es-to-os-migration-b0efd2626f59d977.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes the bulk API of CloudKitty so that it now supports the migration
+    from Elasticsearch to OpenSearch.
```
Comment (markgoddard) [inline releasenotes/notes/cloudkitty-fix-es-to-os-migration-b0efd2626f59d977.yaml:5]: Do we need a new cloudkitty image?

**Review by markgoddard — COMMENTED** [review]
Do we need a neutron fork in Zed+A?

File: releasenotes/notes/cloudkitty-fix-es-to-os-migration-b0efd2626f59d977.yaml (hunk 1 above)
Comment (MoteHue) [reply releasenotes/notes/cloudkitty-fix-es-to-os-migration-b0efd2626f59d977.yaml:5]: Yes we will need to build for both Zed and Antelope.

**Comment by MoteHue** [comment]
> Do we need a neutron fork in Zed+A?

Yes

File: releasenotes/notes/cloudkitty-fix-es-to-os-migration-b0efd2626f59d977.yaml (hunk 1 above)
Comment (MoteHue) [reply releasenotes/notes/cloudkitty-fix-es-to-os-migration-b0efd2626f59d977.yaml:5]: PRs here to bring in the fixes:
https://github.com/stackhpc/cloudkitty/pull/62
https://github.com/stackhpc/cloudkitty/pull/63

**Comment by MoteHue** [comment]
Neutron Zed build: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6863112188
CloudKitty Zed build: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6863132014

**Comment by MoteHue** [comment]
Promotions for cloudkitty running here: https://github.com/stackhpc/stackhpc-release-train/actions/runs/6864671528 https://github.com/stackhpc/stackhpc-release-train/actions/runs/6864674669

**Review by MoteHue — APPROVED** [review]

**Comment by markgoddard** [comment]
> Neutron Zed build: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6863112188 CloudKitty Zed build: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6863132014

I assume you're planning to update neutron separately?

**Comment by MoteHue** [comment]
> I assume you're planning to update neutron separately?

Yes, neutron needs this to be merged upwards into Z and A before they can be built https://github.com/stackhpc/stackhpc-kayobe-config/pull/775

**Comment by markgoddard** [comment]
> > I assume you're planning to update neutron separately?
> 
> Yes, neutron needs this to be merged upwards into Z and A before they can be built #775

Here you go: https://github.com/stackhpc/stackhpc-kayobe-config/pull/783/files

**Pull request merged**